### PR TITLE
Mutliple actions switch example

### PR DESCRIPTION
**Pull request by Letreguilly**
The goal of this pull request is to add two example which performs simple and double press actions on a switch instead of a button. This behaviour can be achieved in the UI for a button but not for a switch as far as I know. The `double-press-double-switch` script has too a special case when the two button are press simultaneously.

### CODE REVIEW

File: double-press-double-switch.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+let doubleClickAction1 = 'http://shelly1-ip/rpc/Switch.Toggle?id=1';
+let simpleClickAction2 = 'http://shelly2-ip/rpc/Switch.Toggle?id=0';
+let doubleClickAction2 = 'http://shelly2-ip/rpc/Switch.Toggle?id=1';
+
+let simpleClickAction1FullOff = 'http://shelly1-ip/rpc/Switch.Set?id=0&on=false';
+let doubleClickAction1FullOff = 'http://shelly1-ip/rpc/Switch.Set?id=1&on=false';
+let simpleClickAction2FullOff = 'http://shelly2-ip/rpc/Switch.Set?id=0&on=false';
+let doubleClickAction2FullOff = 'http://shelly2-ip/rpc/Switch.Set?id=1&on=false';
+
+let doubleClickDelay = 250;
```
Comment (Mircho) [inline double-press-double-switch.js:17]: The standard in our scripts is to have a CONFIG object where you define all the script configs.
It allows for users to locate the place where they can change device configuration

File: double-press-double-switch.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+// the script show how perform mutliple action with a double classic switch
```
Comment (Mircho) [inline double-press-double-switch.js:1]: Can you provide a bit more description of the use case?

File: double-press-double-switch.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        },
+        function (response, error_code, error_message, ud) {
+        },
+        null
+    );
+}
+
+Shelly.addEventHandler(
+    function (event, user_data) {
+        //print(JSON.stringify(event));
```
Comment (Mircho) [inline double-press-double-switch.js:43]: This is not needed

File: double-press-double-switch.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+            } else if (timer === undefined && event.info.id === buttonId2) {
+                
+                timer = Timer.set(doubleClickDelay, 0, toggleLight, simpleClickAction2);
+                previouHitButtonId = buttonId2;
+
+            } else if (timer !== undefined && event.info.id === previouHitButtonId) {
+                if (event.info.id === buttonId1) {
+
+                    reset();
```
Comment (Mircho) [inline double-press-double-switch.js:59]: can you rename to resetTimer

File: double-press-double-switch.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+function toggleLight(action) {
+    reset();
+
+    Shelly.call(
+        "http.get", {
+            url: action
+        },
+        function (response, error_code, error_message, ud) {
```
Comment (Mircho) [inline double-press-double-switch.js:35]: if you don't needed you can omit the callback and the user data parameters

File: double-press-switch.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
+// this script how a classic switch (not a push switch) can be used to to perform different action simple and double
```
Comment (Mircho) [inline double-press-switch.js:1]: Similar comments apply as for the double-press-double-switch.js

**Review by Mircho — COMMENTED** [review]
Thanks for the contribution! Great addition to the functionalities for our users.
Few nitpicks only from my side

File: double-press-double-switch.js (hunk 1 above)
Comment (Letreguilly) [reply double-press-double-switch.js]: I added CONFIG object for the two scripts

File: double-press-double-switch.js (hunk 2 above)
Comment (Letreguilly) [reply double-press-double-switch.js]: description updated with more details

File: double-press-double-switch.js (hunk 3 above)
Comment (Letreguilly) [reply double-press-double-switch.js]: removed

File: double-press-double-switch.js (hunk 4 above)
Comment (Letreguilly) [reply double-press-double-switch.js]: Done

File: double-press-switch.js (hunk 6 above)
Comment (Letreguilly) [reply double-press-switch.js]: Also done

**Pull request merged**